Makefile

```diff
@@ -0,0 +1,75 @@
+#Python3 and virtual environment
+PY3 := $(shell command -v python3 2> /dev/null)
```

Comment:
I don't think this expansion is actually necessary?  If PY3 isn't used as a rule, and rather "assume that python3 is in path" is used, you'll probably have an ok result